.azure/pipelines/quarantined-tests.yml

```
@@ -7,7 +7,6 @@ schedules:
   displayName: Run tests once a day at 11 PM
   branches:
     include:
```

```suggestion
    include:
    - main
```
Missed something in c925f99cddac / #29505